src/firewall/core/io/ipset.py

```diff
@@ -430,6 +430,16 @@ def ipset_reader(filename, path):
             )
     del handler
     del parser
+
+    if "maxelem" in ipset.options and len(ipset.entries) > int(
+        ipset.options["maxelem"]
+    ):
+        raise FirewallError(
+            errors.INVALID_IPSET,
+            "the number of ipset entries exceeds the maximum set size of %s"
+            % ipset.options["maxelem"],
+        )
+
```

Comment:
I think this more correctly belongs in `_check_config()`.

Reply:
You are correct. I have made the necessary changes and modified gh881 maximum element limit.